test/unit/util/test_matrix_policy.hpp

```diff
@@ -204,6 +205,32 @@ MatrixPolicy<double> testAssignmentFromVector()
   return matrix;
 }
 
+template<template<class> class MatrixPolicy>
+MatrixPolicy<double> testAxpy()
+{
+  MatrixPolicy<double> matrix{ 4, 3, 100.0 };
+  MatrixPolicy<double> other{ 4, 3, 200.0 };
```

Comment:
```suggestion
  std::size_t num_rows = 4;
  std::size_t num_columns = 3;
  MatrixPolicy<double> matrix{ 4, 3, 100.0 };
  MatrixPolicy<double> other{ 4, 3, 200.0 };
```
That way we can reuse the dimensions below.

Reply:
I like your suggestion. Done